cost-analyzer/values.yaml

```diff
@@ -2235,6 +2235,46 @@ kubecostDeployment:
   labels: {}
   annotations: {}
 
+## Kubecost Forecasting forecasts future cost patterns based on historical
+## patterns observed by Kubecost.
+forecasting:
+  enabled: false
+
+  # fullImageName overrides the default image construction logic. The exact
+  # image provided (registry, image, tag) will be used for the forecasting
+  # container.
+  # Example: fullImageName: gcr.io/kubecost1/forecasting:v0.0.1
+  fullImageName: gcr.io/kubecost1/kubecost-modeling:6ff46def0575545eb37c5dae179e5035a1de279f
+
+  # Resource specification block for the forecasting container.
+  resources: {}
```

Comment:
Are there truly no resources we can offer as defaults here?

Reply:
A good point. I took a stab at some, let me know if they seem reasonable